python/ray/rllib/models/catalog.py

```diff
@@ -39,6 +39,8 @@
     "fcnet_activation": "tanh",
     # Number of hidden layers for fully connected net
     "fcnet_hiddens": [256, 256],
+    # The dropout rate range from [0, 1]. 0.5 would drop out 50% of hidden units
+    "fcnet_dropout_rate": 0.,
```

Comment:
Is this a common use case? I'm wondering if we can make this PR an example instead. That is, make an example of a using a custom model in `examples/fcnet_with_dropout.py`.

Reply:
The [paper](http://www.jmlr.org/papers/volume15/srivastava14a/srivastava14a.pdf?utm_content=buffer79b43&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer) has ~11500 citations, so I'm assuming that it's usage is quite common. There are already examples of custom networks, so `examples/fcnet_with_dropout.py` might be not really necessary?